lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1166,8 +1166,10 @@ ClangImporter::create(ASTContext &ctx,
   auto fileMapping = getClangInvocationFileMapping(ctx);
   // Wrap Swift's FS to allow Clang to override the working directory
   llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> VFS =
-      llvm::vfs::RedirectingFileSystem::create(
```

Comment:
Is the comment above still true if there are no redirected files?

Reply:
I didn't actually notice that comment. I guess it is to make sure that the clang instance here will definitely get a different FS so clang can call `setWorkingDir` on the FS without interfering with swift? Let me give this more thoughts.